tools/gfx-unit-test/existing-device-handle-test.cpp

```diff
@@ -100,7 +100,7 @@ namespace gfx_test
     {
         if ((api & context->enabledApis) == 0)
```

Comment:
You can use the new runTestImpl function to eliminate all the device creation logic here. See the current compute-smoke test for an example.

Reply:
I'm not seeing that function anywhere. Where is it supposed to be?

Reply:
Ah, never mind, I just realized that it is still in my local branch and I haven't committed it to master yet.

Reply:
Ah, okay. I'll make this change in a later PR once it is then.